Tests/Unit/psgithub.tests.ps1

```diff
@@ -0,0 +1,69 @@
+#
+# This is a PowerShell Unit Test file.
+# You need a unit test framework such as Pester to run PowerShell Unit tests. 
+# You can download Pester from http://go.microsoft.com/fwlink/?LinkID=534084
+#
+
+$Global:ModuleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
```

Comment:
1. I completely agree (global is bad), except in this case when we issue the InModuleScope { } Pester block, the script and local variables are no longer available to us. So the $ModuleRoot is no longer declared. Could redeclare the variable inside the InModuleScope block though.
2. Absolutely :+1: